client/editor/tool_tile.cpp

```diff
@@ -56,14 +62,59 @@ void editor_tool_tile::set_default_values()
   // Set default values and the font we want.
   ui.value_continent->setText(_("-"));
   ui.value_continent->setFont(value_font);
```

Comment:
I didn't notice before that you were using the chat font. I don't see a need for changing it at all, could we use the standard one?